.github/CONTRIBUTING.md

```diff
@@ -0,0 +1,26 @@
+# Contributing Guide
+
+Hello, and thank you for contributing to UltraCosmetics!
+
+Before submitting your contribution, please make sure to take a moment and read through the following guidelines:
+
+- [Code of Conduct](#code-of-conduct)
+- [Issue Reporting Guidelines](#issue-reporting-guidelines)
+- [Pull Request Guidelines](#pull-request-guidelines)
+
+---
+
+## Code of Conduct
+
+Please read and follow our [Code of Conduct](CODE_OF_CONDUCT.md).
+
+## Issue Reporting Guidelines
+
+- When creating an issue, please make sure you're using the correct [issue template](https://github.com/datatags/UltraCosmetics/issues/new/choose).
+
+## Pull Request Guidelines
+
+- When submitting a pull request, please make sure you're using the [pull request template](PULL_REQUEST_TEMPLATE.md).
+- Do not submit pull requests that modify existing strings in the messages files.
+  Changes to those strings should be done through [Crowdin](https://crowdin.com/project/ultracosmetics), because editing them manually can cause Crowdin to lose translations.
+  Adding new strings to the messages files is allowed.
```

Comment:
Is this correct? I believe the previous text ("do not submit pull requests that touch any messages files") was wrong/misleading, right?

Reply:
Yep, that looks good